pom.xml

```diff
@@ -255,6 +255,11 @@
                 <artifactId>log4j</artifactId>
                 <version>1.2.17</version>
             </dependency>
+            <dependency>
```

Comment:
Why ... add LZ4 dependency here?

Reply:
I couldn't build IoTDB due to an error of the dependency plugin ... whithout this there were two different version versions in the reactor.

Reply:
OK, I checked the dependency tree, spark-core uses LZ4 v1.4  while others (e.g., tsfile) use v1.3